### ISSUE TITLE
Address field should be part of the map field instead of separate 

### ISSUE THREAD

**Issue by funkedgeek** [rilwis/meta-box#749]
It'd make cloning so much easier and to me it'd make more sense.

**Comment by Sinusite** [comment]
I Agree :) like https://developers.google.com/maps/documentation/javascript/examples/places-searchbox?hl=fr

Please see https://github.com/rilwis/meta-box/issues/820
